src/Api/Controllers/AppsController.php

```diff
@@ -60,6 +61,19 @@ public function index($id = null): Response
             return $this->getById($id);
```

Comment:
@rwhite27 make the function by id or key that way you just call 1 function and have the same results